docs/source/Participate/PlatformIO.rst

```diff
@@ -81,6 +82,7 @@ Optional, but highly recommended:
 * reStructuredText Syntax highlighting (by Trond Snekvik)
 * Extension pack for reStructuredText (by LeXtudio Inc.)
 * Markdown All in One (by Yu Zhang)
+* WSL (by Microsoft) (when using WSL2 as documented below)
```

Comment:
"WSL2", as you otherwise cannot use USB  pass through.

Reply:
The VSCode extension is named 'WSL', AFAICS.

Reply:
OK, then leave it :)